dev/CommonStyles/CommonStyles.vcxitems

```diff
@@ -42,6 +42,11 @@
       <Version>RS1</Version>
       <Type>ThemeResources</Type>
     </Page>
+    <Page Include="$(MSBuildThisFileDirectory)InkToolbar_themeresources.xaml" >
+	    <UseVisualStyle>Latest</UseVisualStyle>
```

Comment:
Likely you are still in an old version, and you need to merge with master.
UseVisualStyle is renamed
```
 <ControlsResourcesVersion>Version2</ControlsResourcesVersion>
```